### ISSUE TITLE
Why we do not have torch.nn.ReflectionPad3d?

### ISSUE THREAD

**Issue by John1231983** [pytorch/pytorch#27655]
We have used ReflectionPad2d for cyclegan and it is very important to solve the checkboard artifact. 

Considering the 5D case, I also meet the checkboard artifact with cyclegan. I try to find the ReflectionPad3d with the newest PyTorch version but it still in the checklist. Could you tell me when the function will available? It is very useful if it can integrate it in PyTorch

**Comment by zou3519** [comment]
According to the documentation, we do have it: https://pytorch.org/docs/master/nn.html#torch.nn.ReplicationPad3d. What version of PyTorch are you using?

**Comment by John1231983** [comment]
@zou3519 : Maybe you typo. I am asking ReflectionPad3d, instead of ReplicationPad3d? I am using the newest version 1.2.0

**Comment by zou3519** [comment]
Oh you're right, sorry for that. I don't think anyone is working on this.

**Comment by GinSoda** [comment]
Can't agree more, we need ReflectionPad3d!!!

**Comment by NikoOinonen** [comment]
Any update on this? It would be really great to also have 3D reflective padding. I arrived here after trying to use reflective padding with a Conv3D layer and getting a ```NotImplementedError```, because the documentation for Conv3D incorrectly lists ```'reflect'``` as a valid padding mode.

**Comment by jerofad** [comment]
Is there a way to extend the nn.ReflectionPad2d to work for 3d ?
